GitUI/CommandsDialogs/FormDeleteRemoteBranch.cs

```diff
@@ -110,13 +133,21 @@ private void Delete_Click(object sender, EventArgs e)
                 if (!form.ErrorOccurred() && !Module.InTheMiddleOfAction())
                 {
                     ScriptsRunner.RunEventScripts(ScriptEvent.AfterPush, this);
+                    if (DeleteLocalTrackingBranch.Checked)
```

Comment:
I don't think the checkbox should be enable if there are no tracking refs available....